config/application.rb

```diff
@@ -107,113 +108,59 @@ def config.database_configuration
     # issue is image_optim crashes on missing dependencies
     config.assets.image_optim = false
 
-    config.autoloader = :zeitwerk
+    require 'discourse_inflector'
+    Rails.autoloaders.each do |autoloader|
+      autoloader.inflector = DiscourseInflector.new
+      autoloader.inflector.inflect(
```

Comment:
How hard would it be to fix those as well?

Reply:
Sorry, forgot to answer your comment! :weary: Fixing these should be “just” (:crossed_fingers:) renaming and tracking where the constants are used.